src/parser.ts

```diff
@@ -1,11 +1,11 @@
 import { CstParser } from 'chevrotain';
-import getConfig from './getConfig';
+import { PrismaAstConfig } from './getConfig';
 import * as lexer from './lexer';
 
 type ComponentType = 'datasource' | 'generator' | 'model' | 'view' | 'enum';
 export class PrismaParser extends CstParser {
-  constructor() {
-    super(lexer.multiModeTokens, getConfig().parser);
+  constructor(config: PrismaAstConfig) {
+    super(lexer.multiModeTokens, config.parser);
```

Comment:
In this context we can scope the param to just the parser portion of the config

```suggestion
import getConfig, { type PrismaAstConfig } from './getConfig';
type ParserOptions = PrismaAstConfig['parser'];
// ...

  constructor(options: ParserOptions = getConfig().parser) {
    super(lexer.multiModeTokens, config.parser);
```